client/allocrunner/taskrunner/envoy_bootstrap_hook_test.go

```diff
@@ -516,7 +518,7 @@ func TestTaskRunner_EnvoyBootstrapHook_gateway_ok(t *testing.T) {
 
 	// the only interesting thing on bootstrap is the presence of the cluster,
 	// everything is configured at runtime through xDS
-	require.Equal(t, "my-ingress-service", out.Node.Cluster)
+	require.Equal(t, "ingress-gateway", out.Node.Cluster)
```

Comment:
Seems to be a quirk of Consul, if you pass `-proxy-id` is uses `ingress-gateway` as the envoy cluster name, otherwise the service name

Reply:
Do we know if it's a bug or deliberate? Seems like it'd run into collisions too.

Reply:
Looked closer, it's because it can namespace on `id` now, e.g. 

```
  "node": {
    "cluster": "ingress-gateway",
    "id": "_nomad-task-85e65316-c6f9-44ba-3bc1-ea8cd2dfd493-group-ingress-group-my-ingress-service-8080",
    "metadata": {
      "namespace": "default",
      "envoy_version": "1.16.0"
    }
  },
```

Reply:
Guess I should add that to the test